packages/shared/src/components/auth/OnboardingSteps/ContentTypes/ContentTypes.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import useFeedSettings from '../../../../hooks/useFeedSettings';
+
+import { useAdvancedSettings } from '../../../../hooks';
+import { CustomCheckbox } from './CustomCheckbox';
+import { useContentTypesOnboarding } from './useContentTypesOnboarding';
+
+export const ContentTypes = (): ReactElement => {
+  const { contentSourceList } = useFeedSettings();
+  const {
+    selectedSettings,
+    onToggleSettings,
+    checkSourceBlocked,
+    onToggleSource,
+  } = useAdvancedSettings();
+  const { advancedSettingsExceptCommunity } = useContentTypesOnboarding();
```

Comment:
I'm just confused why we need this, you can just use `contentCurationList` from the above hook right?

Reply:
yes, it's removed